app/design/frontend/Magento/blank/web/css/source/_reset.less

```diff
@@ -4,5 +4,7 @@
 //  */
 
 & when (@media-common = true) {
+    .lib-html5doctor-reset();
```

Comment:
This feels to me like we're including a lot of properties that we probably don't actually need. What aspects of this reset are not already being met by the generic `lib-magento-reset` mixin below it?

This line for instance: 

https://github.com/magento/magento2/blob/7c6b6365a3c099509d6f6e6c306cb1821910aab0/lib/web/css/source/lib/_resets.less#L439-L442

Is mostly a dupe of this block, from `lib-magento-reset`:

https://github.com/magento/magento2/blob/7c6b6365a3c099509d6f6e6c306cb1821910aab0/lib/web/css/source/lib/_resets.less#L20-L32

Perhaps rather than calling this mixin directly, add the needed lines from the html5doctor mixin to the generic reset mixin instead?

Reply:
Thank you for your comment!